client/src/Components/Common/NavTap/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import './style.css';
+import { Radio } from 'antd';
+import { ReactComponent as Home } from '../../../assets/icons/home.svg';
+import { ReactComponent as About } from '../../../assets/icons/about.svg';
+import { ReactComponent as Search } from '../../../assets/icons/search.svg';
+
+const NavTap = () => (
+  <Radio.Group buttonStyle="solid">
+    <Radio.Button value="1">
+      <Home className="home" />
+      الرئيسية
+    </Radio.Button>
+    <Radio.Button value="2">
+      <About className="about" />
+      حول
+    </Radio.Button>
+    <Radio.Button value="3">
+      <Search className="search" />
+      البحث
+    </Radio.Button>
+  </Radio.Group>
```

Comment:
Instead of using the Radio Button try to use Menu, it's better

```suggestion
  <Menu mode="horizontal" defaultSelectedKeys={['1']}>
    <Menu.Item key="1" icon={<Home className="home" />}>
      الرئيسية
    </Menu.Item>
    <Menu.Item key="2" icon={<About className="about" />}>
      حول
    </Menu.Item>
    <Menu.Item key="3" icon={<Search className="search" />}>
      البحث
    </Menu.Item>
  </Menu>
```